ssn/integrated/oldssn.ttl

```diff
@@ -530,24 +530,6 @@ ssn:Drift rdf:type owl:Class ;
           rdfs:seeAlso "http://www.w3.org/2005/Incubator/ssn/wiki/SSN_Sensor#MeasuringCapability" .
 
 
-###  http://www.w3.org/ns/ssn/FeatureOfInterest
```

Comment:
removed ssn:FeatureOfInterest in the oldssn.ttl, according to the proposed integration methodology